docker-compose.yml

```diff
@@ -6,6 +6,8 @@ services:
     build:
       context:  ./
       dockerfile: Dockerfile-frankyapp
+    env_file:
+      - .env
```

Comment:
Fix me if I wrong but these configuration can be omitted. `.env` is a default environment file.

Reply:
dont work without this lines

Reply:
Ok, lets leave it as is.